### PR TITLE
Add subcatchment surface buildup to subcatchment statistics

### PR DESCRIPTION
**Pull request by katmratliff**
Completes #125. Updated subcatchment flow stats to include `pollutant_buildup` (subcatchment surface buildup, reported in lbs or kg per ac or ha depending on `sim.system_units`).

Example below includes variable initial pollutant buildup (10, 20, 30 lbs/acre), a max buildup of 40 lbs/acre, linear buildup function of 5 lbs/day, and a rain event:
sc_buildups

### CODE REVIEW

**Comment by bemcdonnell** [comment]
@katmratliff, does this PR have the new binary from Anaconda? The only test that is failing is the recent additions that I made a few weeks ago (leading me to believe that the DLL does not have the new function. I'm trying to isolate if those changes I made are actually in that swmm build

File: pyswmm/tests/test_pollutants.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014 Bryant E. McDonnell
```
Comment (bemcdonnell) [inline pyswmm/tests/test_pollutants.py]: @katmratliff, you can put your info here instead of mine

File: pyswmm/subcatchments.py [hunk 2]
```diff
@@ -574,6 +574,8 @@ def statistics(self):
         +---------------------+
         | Peak Runoff Rate    |
         +---------------------+
+        | Pollutant Buildup   |
```
Comment (bemcdonnell) [inline pyswmm/subcatchments.py]: @katmratliff, so regarding these rST tables, would you mind doing me a favor?  I think we should list the actual dictionary keys (this is in Nodes, Links, System, and Subatchments modules).  If you don't want to do it, I can :-)  It's my fault afterall :-D

This table generator is really nice: 

https://www.tablesgenerator.com/text_tables

**Comment by katmratliff** [comment]
@bemcdonnell before submitting the pull request I merged all of the pyswmm updates except for the DLL because I knew I was going to be updating that anyways. The DLL is from recompiling my SWMM repository so it might not be up-to-date... I'll check on that now

**Comment by katmratliff** [comment]
@bemcdonnell I'm having some trouble installing the ciocheck package... getting an error during installation via conda (`CondaError: Cannot link a source that does not exist. C:\Users\kratliff\AppData\Local\Continuum\Anaconda3\Scripts\conda.exe`). Is there another way to install other than conda?

**Comment by bemcdonnell** [comment]
@goanpeca, does @katmratliff just need to make sure this is on the path?  I thought that ciocheck was part of anaconda

**Comment by bemcdonnell** [comment]
@katmratliff, as an example mine is sitting in this directory `C:\Users\Bryant\Anaconda3\Scripts`

**Comment by katmratliff** [comment]
@bemcdonnell ciocheck didn't come as part of my anaconda installation. I can try reinstalling

**Comment by bemcdonnell** [comment]
https://anaconda.org/conda-forge/ciocheck 

`conda install -c conda-forge ciocheck`

**Comment by katmratliff** [comment]
@bemcdonnell yeah, that's how i've been trying to install, but I get this feedback: 

C:\Users\kratliff>C:\Users\kratliff\AppData\Local\Continuum\Anaconda3\Scripts\conda install -c conda-forge ciocheck
Fetching package metadata .............
Solving package specifications: .

Package plan for installation in environment C:\Users\kratliff\AppData\Local\Continuum\Anaconda3:

The following NEW packages will be INSTALLED:

    apipkg:          1.4-py36_0    conda-forge
    autopep8:        1.3.3-py_0    conda-forge
    ca-certificates: 2017.7.27.1-0 conda-forge
    ciocheck:        0.1.1-py36_1  conda-forge
    coverage:        4.4.2-py36_0  conda-forge
    execnet:         1.5.0-py_0    conda-forge
    flake8:          3.3.0-py36_0  conda-forge
    mccabe:          0.6.1-py36_0  conda-forge
    pycodestyle:     2.3.1-py36_0  conda-forge
    pydocstyle:      2.0.0-py36_0  conda-forge
    pytest-cov:      2.5.1-py36_0  conda-forge
    pytest-forked:   0.2-py36_0    conda-forge
    pytest-json:     0.4.0-py36_0  conda-forge
    pytest-xdist:    1.20.1-py_0   conda-forge
    vc:              14-0          conda-forge
    yapf:            0.18.0-py_0   conda-forge

The following packages will be UPDATED:

    conda:           4.3.21-py36_0             --> 4.3.29-py36_0 conda-forge

The following packages will be SUPERSEDED by a higher-priority channel:

    conda-env:       2.6.0-0                   --> 2.6.0-0       conda-forge
    openssl:         1.0.2l-vc14_0             --> 1.0.2l-vc14_0 conda-forge [vc14]

Proceed ([y]/n)? y

ERROR conda.core.link:_execute_actions(337): An error occurred while installing package 'conda-forge::yapf-0.18.0-py_0'.
CondaError: Cannot link a source that does not exist. C:\Users\kratliff\AppData\Local\Continuum\Anaconda3\Scripts\conda.exe
Attempting to roll back.


CondaError: Cannot link a source that does not exist. C:\Users\kratliff\AppData\Local\Continuum\Anaconda3\Scripts\conda.exe

**Comment by katmratliff** [comment]
@stoiver I had changed my path before and got the same error. Just changed it back again and then ran activate.bat and still getting same error.

**Comment by bemcdonnell** [comment]
@katmratliff, I can run ciocheck for you just so we can determine if the tests pass or fail.

**Comment by goanpeca** [comment]
@katmratliff it could be an install issue.

Maybe you could reinstall but using a path like 

`C:\Users\kratliff\Anaconda3`

Also, make sure you are using the `Anaconda Prompt`

**Comment by bemcdonnell** [comment]
@katmratliff, please merge my PR into your branch.  I ran ciocheck and i updated the SWMM lib.. For some reason the version on anaconda doesn't seem to have my boundary condition changes.

**Comment by katmratliff** [comment]
@goanpeca still get the same `CondaError` when using `Anaconda Prompt`. I'll reinstall at some point and test again.

File: pyswmm/subcatchments.py (hunk 2 above)
Comment (katmratliff) [reply pyswmm/subcatchments.py]: @bemcdonnell hopefully I caught them all!

**Comment by bemcdonnell** [comment]
@katmratliff, in the docs there is an authors file as well. Could you please add your name to the maintainers block?
